demos/drawer/drawer.scss

```diff
@@ -59,36 +60,48 @@
   padding: 10px;
 }
 
-.demo-drawer--custom {
-  @include mdc-drawer-fill-color($material-color-teal-900);
-  @include mdc-drawer-ink-color(#fff);
-  @include mdc-list-item-graphic-ink-color(#fff);
+.demo-form-field {
+  margin: 16px 0;
+}
 
-  .demo-drawer-list-item {
-    @include mdc-list-item-graphic-ink-color(text-icon-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-secondary-on-dark);
-  }
+/* Place drawer and content side by side. */
+.demo-body {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+}
 
-  .demo-drawer-list-item.mdc-list-item--selected {
-    @include mdc-list-item-graphic-ink-color(text-primary-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-primary-on-dark);
-  }
+.demo-content {
+  height: 100%;
+  width: 100%;
+  overflow: auto;
 }
 
-.demo-drawer--accessible {
-  @include mdc-drawer-fill-color-accessible($material-color-indigo-500);
+.demo-main {
+  padding-left: 16px;
+  padding-right: 16px;
+}
 
-  .demo-drawer-list-item {
-    @include mdc-list-item-graphic-ink-color(text-icon-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-secondary-on-dark);
-  }
+#extra-wide-content {
+  width: 200vw;
+}
 
-  .demo-drawer-list-item.mdc-list-item--selected {
-    @include mdc-list-item-graphic-ink-color(text-primary-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-primary-on-dark);
-  }
+#extra-tall-content {
+  height: 200vh;
 }
 
-.demo-form-field {
-  margin: 16px 0;
+.demo-drawer--custom {
+  @include mdc-drawer-border-color(lightblue, $mdc-drawer-divider-opacity);
+  @include mdc-drawer-group-header-text-color(darkolivegreen, $mdc-drawer-subtitle-ink-opacity);
+  @include mdc-drawer-surface-fill-color(pink, $mdc-drawer-surface-fill-opacity);
+  @include mdc-drawer-icon-ink-color(darkolivegreen, $mdc-drawer-icon-ink-opacity);
+  @include mdc-drawer-title-ink-color(darkolivegreen, 1);
+  @include mdc-drawer-activated-overlay-color(darkcyan);
+  @include mdc-drawer-item-text-color(darkolivegreen, $mdc-drawer-inactive-ink-opacity);
```

Comment:
Is there a reason we seem to have mixins explicitly targeting _only_ the text and _only_ the icon?

Shouldn't there be plenty of cases where both have the same color? Can we target both text and icon in one mixin for the simple case, then have another specifically for overriding just the icon? That way if you want to customize both to the same color, you only have to call one mixin.

(Technically the `mdc-list-item-primary-text-ink-color` mixin doesn't restrict its selector, so it can be used for this purpose... maybe we should eventually rename it to reflect that. I guess this whole comment also ties back to the duplication of mixins between list and drawer...)

Reply:
I like this suggestion - I will update the text-ink-color to include graphic and meta mixins

Reply:
Does it actually need those in order to work? I was hoping they would Just Work based on simply setting the color of the list item as a whole. The point was to both reduce excess Sass _and_ output CSS.

Reply:
I thought we were going to do this change so the user didn't need 2 mixins (`mdc-drawer-item-text-color` and `mdc-drawer-icon-ink-color`), but instead just 1 which incapsulates both the text and icon colors.

Reply:
Spoke offline - because list sets graphic and meta colors in baseline styles we can't just use the primary-text-color mixin for all list item colors. We are keeping them as separate mixins.